pkg/kubectl/cmd/util/shortcut_restmapper.go

```diff
@@ -67,7 +69,19 @@ func (e ShortcutExpander) getAll() []schema.GroupResource {
 		}
 	}
 
-	return availableAll
+	// populate all "discovered" resources from the server
+	for availableResource := range availableResources {
+		r := availableResource.GroupResource()
```

Comment:
AFAIK, range slice will return index if only one lvalue, and int did not have `GroupResource` funcs. may be:

```
  for _, availableResource := range availableResources {
    ...
  }
```